docs/pages/setup/reference/cli.mdx

```diff
@@ -743,6 +745,58 @@ add_headers:
       "Authorization": "Bearer tokentokentoken"
 ```
 
+#### Aliases
+
+Aliases allow you to define custom commands or change the default flag values for existing commands.
+
+New command `tsh l`:
+
+```yaml
+aliases:
+    "l": "tsh login --auth=okta --username=alice"
+```
+
+Make `tsh status` use JSON as a default format:
+
+```yaml
+aliases:
+    "status": "tsh status --format=json"
+```
+
+Define custom command using `bash`. The `$0` and `$1` variables will be substituted with command arguments.
+
+```yaml
+aliases:
+    "connect": "bash -c 'tsh login $0 && tsh ssh $1'"
+```
+
+The alias can use an arbitrary number of arguments. If a argument variable `$N` is referenced, `tsh` will check that at least `N+1` arguments were given to the alias invocation. All arguments that were given but not referenced in the alias definition will be appended at the end.
```

Comment:
- I would move this general explanation of the logic for checking and passing parameters above the "Define custom command" example. Otherwise, I got a bit confused as to how that example works.

- Does `tsh` only apply parameter checking/passing logic to `bash -c` aliases, or does it look for any `$[0-9]+` in an alias? What happens if there are numbered variables but no invocation of `bash` (or another shell that supports numbered variables)?